website/content/docs/developers/simulator-mode.md

```diff
@@ -39,8 +39,17 @@ simulator.deactivate("test");
 simulator.shutdown();
 ```
 
-To kill a topology, you could also simply terminate this process.
+To kill a topology, one could also terminate the process.
+
+The simulator mode will run in separate threads other than main thread. All those interfaces are thread-safe. This means that one could invoke these interfaces in other threads and monitor the corresponding behaviors interactively.
+
+# Debug topology using IntelliJ
+
+Bolts and Spouts run as separate threads in simulator. To add breakpoints inside a bolt/spout, the Suspend Policy of the breakpoint needs to be set as Thread. To change the Suspend Policy, Right click on the breakpoint as shown in the following image:
```

Comment:
Lowercase 'right click'.

Also here and elsewhere best to line break these at 80 or 100 to make it easier to provide comments and read in an IDE.